ui/v2.5/src/components/Tagger/queries.ts

```diff
@@ -204,6 +208,54 @@ export const useUpdateStudioStashID = () => {
   return handleUpdate;
 };
 
+export const useUpdateStudio = () => {
+  const [updateStudio] = GQL.useStudioUpdateMutation({
+    onError: (errors) => errors,
+    errorPolicy: "all",
+  });
+
+  const updateStudioHandler = (input: GQL.StudioUpdateInput) =>
+    updateStudio({
+      variables: {
+        input,
+      },
+      update: (store, updatedStudio) => {
+        if (!updatedStudio.data?.studioUpdate) return;
+
+        if (updatedStudio.data?.studioUpdate?.parent_studio) {
+          const ac = getClient();
+          ac.refetchQueries({ include: studioMutationImpactedQueries });
```

Comment:
The goal of this was that if a parent was created/updated, since the data for that studio was not being passed back to update the cache directly, the whole thing should be refreshed.
Perhaps there is a better way of doing this.